### ISSUE TITLE
'-w' does not work on eslint-watch@2.1.10

### ISSUE THREAD

**Issue by takajun** [rizowski/eslint-watch#79]
We always thank for your great work.

I upgraded eslint-watch from 2.1.7 to 2.1.10 under windows to use eslint>2.0.0, then got the error with '-w' option.
Could you check it?
eslint-watch@2.1.10, eslint@2.4.0 or eslint@1.10.3

```
Z:\>set DEBUG=esw:* & node_modules\.bin\esw -w test.js
  esw:eslint-cli Loaded +0ms
  esw:eslint-cli EsLint path: Z:\node_modules\.bin\eslint.cmd +16ms
  esw:eslint-help Loaded +47ms
  esw:options Loaded +31ms
  esw:success-formatter loaded +343ms
  esw:simple-detail loaded +0ms
  esw:watcher Loaded +0ms
  esw:arg-parser Loaded +0ms
  esw:esw-cli Loaded +16ms
  esw:esw-cli Eslint-Watch: 2.1.10 +0ms
  esw:eslint-help Executing help +0ms
  esw:eslint-cli eslint: [ '--help' ] +0ms
  esw:eslint-help Help text received +436ms
  esw:eslint-help noAlias +0ms false
  esw:eslint-help Alias found: -c +0ms
  esw:eslint-help Parsing --config +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --no-eslintrc +0ms
  esw:eslint-help Parsing no option +16ms --no-eslintrc
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --env +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --ext +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --global +16ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --parser +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --parser-options +0ms
  esw:eslint-help noAlias +15ms true
  esw:eslint-help Parsing --cache +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --cache-file +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --cache-location +16ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --rulesdir +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --plugin +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --rule +15ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --ignore-path +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --no-ignore +0ms
  esw:eslint-help Parsing no option +16ms --no-ignore
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --ignore-pattern +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --stdin +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --stdin-filename +16ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --quiet +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --max-warnings +0ms
  esw:eslint-help noAlias +0ms false
  esw:eslint-help Alias found: -o +0ms
  esw:eslint-help Parsing --output-file +15ms
  esw:eslint-help noAlias +0ms false
  esw:eslint-help Alias found: -f +0ms
  esw:eslint-help Parsing --format +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --no-color +0ms
  esw:eslint-help Parsing no option +16ms --no-color
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --init +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --fix +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --debug +0ms
  esw:eslint-help noAlias +15ms false
  esw:eslint-help Alias found: -h +0ms
  esw:eslint-help Parsing --help +0ms
  esw:eslint-help noAlias +0ms false
  esw:eslint-help Alias found: -v +0ms
  esw:eslint-help Parsing --version +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --no-inline-config +16ms
  esw:eslint-help Parsing no option +0ms --no-inline-config
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --print-config +0ms
  esw:esw-cli Arguments passed: [ 'C:\\Program Files\\nodejs\\node.exe', 'Z:\\node_modules\\eslint-watch\\bin\\esw', '-w', 'test.js' ] +16ms
  esw:esw-cli Parsing args +0ms
  esw:arg-parser Directories to check: [ 'test.js' ] +0ms
  esw:arg-parser Args [ '-w', 'test.js' ] +0ms
  esw:arg-parser Pushing item: test.js +15ms
  esw:arg-parser setting custom formatter +0ms
  esw:arg-parser GetPath: simple-detail +0ms
  esw:esw-cli Running initial lint +0ms
  esw:esw-cli [ 'test.js', '-f', 'Z:\\node_modules\\eslint-watch\\src\\formatters\\simple-detail' ] +0ms
  esw:eslint-cli eslint: [ 'test.js', '-f', 'Z:\\node_modules\\eslint-watch\\src\\formatters\\simple-detail' ] +0ms
  esw:esw-cli -w seen +16ms
  esw:watcher { useEslintrc: true, ignore: true } +0ms
  esw:watcher { watch: true, format: 'simple-detail', eslintrc: true, ignore: true, color: true, inlineConfig: true, _: [ 'test.js' ] } +15ms
  esw:watcher Watching: [ 'test.js' ] +78ms
  esw:eslint-cli 0 +16ms
{ [Error: watch test.js EINVAL]
  code: 'EINVAL',
  errno: 'EINVAL',
  syscall: 'watch test.js',
  filename: 'test.js' }
  esw:simple-detail loaded +0ms
  esw:simple-detail [ { filePath: 'Z:\\test.js', messages: [], errorCount: 0, warningCount: 0 } ] +15ms
✓ Clean (09:40:52)

  esw:esw-cli exitCode: 0 +530ms
```

**Comment by rizowski** [comment]
@takajun I haven't been able to reproduce this error locally. I have seen a few things around `EINVAL` being related to cygwin. What terminal do you use?

This is what I get when I run the command:
```shell
esw:eslint-cli Loaded +0ms
  esw:eslint-cli EsLint path: C:\Users\rizowski\git\eslint-watch\node_modules\.bin\eslint.cmd +3ms
  esw:eslint-help Loaded +30ms
  esw:options Loaded +26ms
  esw:success-formatter loaded +342ms
  esw:simple-detail loaded +3ms
  esw:watcher Loaded +1ms
  esw:arg-parser Loaded +1ms
  esw:esw-cli Loaded +1ms
  esw:esw-cli Eslint-Watch: 2.1.10 +0ms
  esw:eslint-help Executing help +1ms
  esw:eslint-cli eslint: [ '--help' ] +0ms
  esw:eslint-help Help text received +395ms
  esw:eslint-help noAlias +1ms false
  esw:eslint-help Alias found: -c +1ms
  esw:eslint-help Parsing --config +2ms
  esw:eslint-help noAlias +1ms true
  esw:eslint-help Parsing --no-eslintrc +0ms
  esw:eslint-help Parsing no option +1ms --no-eslintrc
  esw:eslint-help noAlias +1ms true
  esw:eslint-help Parsing --env +0ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --ext +1ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --global +1ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --parser +1ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --parser-options +1ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --cache +1ms
  esw:eslint-help noAlias +1ms true
  esw:eslint-help Parsing --cache-file +0ms
  esw:eslint-help noAlias +1ms true
  esw:eslint-help Parsing --cache-location +0ms
  esw:eslint-help noAlias +1ms true
  esw:eslint-help Parsing --rulesdir +0ms
  esw:eslint-help noAlias +1ms true
  esw:eslint-help Parsing --plugin +0ms
  esw:eslint-help noAlias +1ms true
  esw:eslint-help Parsing --rule +1ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --ignore-path +1ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --no-ignore +1ms
  esw:eslint-help Parsing no option +2ms --no-ignore
  esw:eslint-help noAlias +4ms true
  esw:eslint-help Parsing --ignore-pattern +1ms
  esw:eslint-help noAlias +1ms true
  esw:eslint-help Parsing --stdin +1ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --stdin-filename +1ms
  esw:eslint-help noAlias +1ms true
  esw:eslint-help Parsing --quiet +0ms
  esw:eslint-help noAlias +5ms true
  esw:eslint-help Parsing --max-warnings +1ms
  esw:eslint-help noAlias +1ms false
  esw:eslint-help Alias found: -o +1ms
  esw:eslint-help Parsing --output-file +1ms
  esw:eslint-help noAlias +1ms false
  esw:eslint-help Alias found: -f +0ms
  esw:eslint-help Parsing --format +1ms
  esw:eslint-help noAlias +1ms true
  esw:eslint-help Parsing --no-color +7ms
  esw:eslint-help Parsing no option +1ms --no-color
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --init +1ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --fix +1ms
  esw:eslint-help noAlias +0ms true
  esw:eslint-help Parsing --debug +5ms
  esw:eslint-help noAlias +1ms false
  esw:eslint-help Alias found: -h +0ms
  esw:eslint-help Parsing --help +1ms
  esw:eslint-help noAlias +1ms false
  esw:eslint-help Alias found: -v +0ms
  esw:eslint-help Parsing --version +1ms
  esw:eslint-help noAlias +2ms true
  esw:eslint-help Parsing --no-inline-config +4ms
  esw:eslint-help Parsing no option +0ms --no-inline-config
  esw:eslint-help noAlias +1ms true
  esw:eslint-help Parsing --print-config +0ms
  esw:esw-cli Arguments passed: [ 'C:\\Program Files\\nodejs\\node.exe', 'C:\\Program Files\\nodejs\\node_modules\\eslint-watch\\bin\\esw', '--quiet' ] +5ms
  esw:esw-cli Parsing args +8ms
  esw:arg-parser Directories to check: [] +1ms
  esw:arg-parser Args [ '--quiet' ] +2ms
  esw:arg-parser Pushing item: --quiet +3ms
  esw:arg-parser setting custom formatter +1ms
  esw:arg-parser GetPath: simple-detail +0ms
  esw:arg-parser Setting default path: ./ +1ms
  esw:esw-cli Running initial lint +0ms
  esw:esw-cli [ '--quiet', '-f', 'C:\\Users\\rizowski\\git\\eslint-watch\\src\\formatters\\simple-detail', './' ] +1ms
  esw:eslint-cli eslint: [ '--quiet', '-f', 'C:\\Users\\rizowski\\git\\eslint-watch\\src\\formatters\\simple-detail', './' ] +1ms
  esw:eslint-cli 0 +20ms
  esw:simple-detail loaded +0ms
  esw:simple-detail [] +5ms
✓ Clean (9:18:53 AM)

  esw:esw-cli exitCode: 0 +4s
```

**Comment by takajun** [comment]
@rizowski I use Windows's CommandPrompt as terminal. And I also don't see any error without '-w' option. Just only with '-w' option.
I tried the following:
Windows7 64bit, node v4.4.0: newly npm install eslint-watch, and I got the error the above.
Windows10 64bit(anotherPC), node v4.4.0: newly npm install eslint-watch, and no error.

This issue may be caused by my PC condition/setting, but I don't see what went wrong...
